serve-site.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+cd www
+npx http-server -c-1 
```

Comment:
I'm not sure how useful this is if we plan on switching to a bundler anyway later?

Reply:
I just wanted have _something_ to serve the site locally so others can see how to use this until we switch to a bundler :)